### PR TITLE
12502/Fss-ui-simplified-search-screen-and-text-box-with-add-validation

### PR DESCRIPTION
**Pull request by Rajesh14881**
Created simplified search UI with validation

### CODE REVIEW

**Review by jimjamjahaa — APPROVED** [review]
lgtm

File: src/app/features/fss-search/fss-simplified-search/fss-simplified-search.component.scss [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+	.text-input-wrap{
+		height: initial;
+	}
+	.simplifiedTextbox{
+	    width: 40%;
+		float: left;
+		margin-right: 20px;
+	}
+}
+.switchToAdvanded{
```
Comment (gregzealley) [inline src/app/features/fss-search/fss-simplified-search/fss-simplified-search.component.scss:11]: Advanded ?

**Review by gregzealley — CHANGES_REQUESTED** [review]

**Review by bhalcha13813 — APPROVED** [review]
LGTM

File: Tests/FunctionalTests/fss-simplified-search-home.spec.ts [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+  })
+
+  it('Does it display "Advanced Search" link on Simplified Search page', async () => {
+       
+    var advancedSearchLink = await page.innerText(pageObjectsConfig.advancedSeatchLinkSelector);
+    expect(advancedSearchLink).toEqual(pageObjectsConfig.advancedSeatchLink);
+
+  })
+
+  it('Does it display "Error message" if user clicks on serach button and simplified search box is empty', async () => {
```
[the remark this answers is not in the document]
Comment (kajalkundu) [reply Tests/FunctionalTests/fss-simplified-search-home.spec.ts]: Uplifted the typo error. Thanks.

File: Tests/FunctionalTests/fss-simplified-search-home.spec.ts [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+    await page.waitForSelector(pageObjectsConfig.inputSimplifiedSerachBoxSelector);
```
[the remark this answers is not in the document]
Comment (kajalkundu) [reply Tests/FunctionalTests/fss-simplified-search-home.spec.ts]: Uplifted the typo error. Thanks.

File: Tests/FunctionalTests/pageObjects.json [hunk 4]
```diff
@@ -67,7 +67,14 @@
        "paginatorSelector" : "ukho-paginator p",
        "acceptCookieSelector" : "#onetrust-accept-btn-handler",
        "skipContentSelector" : "a.skip-to-content-link",
-       "warningMessageValue": "There is a problem with a field"
+       "warningMessageValue": "There is a problem with a field",
+       "simplifiedSeatchLinkSelector" : "a.switchToSimplified",
+       "simplifiedSeatchLinkText" : "Simplified Search",
+       "inputSimplifiedSerachBoxSelector" : "app-fss-simplified-search ukho-textinput >input.form-text-input",
+       "advancedSeatchLinkSelector" : "a.switchToAdvanded",
+       "advancedSeatchLink" : "Advanced Search",
```
[the remark this answers is not in the document]
Comment (kajalkundu) [reply Tests/FunctionalTests/pageObjects.json]: Uplifted the typo error. Thanks.

File: src/app/features/fss-search/fss-simplified-search/fss-simplified-search.component.scss (hunk 1 above)
Comment (kajalkundu) [reply src/app/features/fss-search/fss-simplified-search/fss-simplified-search.component.scss]: Change the class name to **.switchToAdvanced** . Thanks.

**Pull request merged**